src/Keboola/DynamoDbExtractor/Extractor.php

```diff
@@ -76,16 +77,22 @@ public function actionRun(string $outputPath): void
                     $parser->parseAndWriteCsvFiles();
                     $export->cleanup();
 
-                    $manifestOptions = new OutTableManifestOptions();
-                    $manifestOptions->setIncremental($exportOptions['incremental']);
+                    $options = new ManifestManager\Options\OutTable\ManifestOptions();
+                    $options
+//                        ->setSchema($this->createSchema($this->tempTable))
+                        ->setIncremental($exportOptions['incremental'])
+//                        ->setDelimiter(Csv::UNLOAD_CSV_DELIMITER)
+//                        ->setEnclosure(Csv::UNLOAD_CSV_ENCLOSURE);
+
```

Comment:
ten ex neřeší vůbec sloupce takže to bude potřeba nějak vytáhnůt z toho dynama, nebo seznam pokud je v configu